autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -44,14 +44,28 @@ def __init__(self,
 
         It also allows to define a 'epoch_or_time' budget type, which means,
         the first of them both which is exhausted, is honored
+        Args:
+            budget_type (str): Type of budget to be used when fitting the pipeline.
```

Comment:
Could you add a new line and indentation to the description of `budget_type`? You can check the base_task.py file for how we format the docstrings.